.github/workflows/build.yml

```diff
@@ -0,0 +1,325 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: "0 7 * * *"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  azCliVersion: 2.6.0
```

Comment:
In future, if we have to update this, where can we find the latest version number?

Reply:
The latest version [is always at this hyperlink](https://docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?tabs=azure-cli)